control/control.leanos.xml

```diff
@@ -139,7 +139,8 @@ textdomain="control"
             </one_supported_desktop>
         </supported_desktops>
 
-        <default_desktop>gnome</default_desktop>
+        # use default desktop order based on installed packages
```

Comment:
⚠️ Parse Error.

Something like `no default desktop, can be redefined by an add-on` sounds better...

Reply:
actually not. This is for https://trello.com/c/NmLDu0Gz/1730-bring-back-icewm

`It is do not define default desktop in control file and use instead update-alternatives approach` . Is it better?

Reply:
OK, even better! 👍

Reply:
After reading again I'd suggest just a better wording: `Do not define the default desktop in control file, use the update-alternatives approach instead`